openvpn-install.sh

```diff
@@ -1128,8 +1096,8 @@ function revokeClient () {
 
 function removeUnbound () {
 	# Remove OpenVPN-related config
-	sed -i 's|include: \/etc\/unbound\/openvpn.conf||' /etc/unbound/unbound.conf
-	rm /etc/unbound/openvpn.conf
+	sed -i '/include .*\/openvpn.conf/d' /etc/unbound/unbound.conf
```

Comment:
Can we use `/etc/unbound/unbound.conf.d/openvpn.conf` instead of wildcard?

Reply:
Yes. It's only longer, would not find the old location `/etc/unbound/openvpn.conf`, and needs lot of backslashes  ;-)
`
sed -i '/include \/etc\/unbound\/unbound.conf.d\/openvpn.conf/d' /etc/unbound/unbound.conf
`

Reply:
I think it's clearer to specify the full path but I'm fine anyway.